openhands/events/replay.py

```diff
@@ -135,7 +135,17 @@ def handle_replay_internal_observation(
             # New workflow: initial-analysis provided the metadata to allow tool use.
             metadata, data = split_metadata(result)
             prefix = ''
-            suffix = f'* This bug had a timetravel debugger recording.\n* Use below `Initial Analysis` and the timetravel debugger `inspect-*` tools to find the bug.\n* Once found, `submit-hypothesis`, so your analysis can be used to implement the solution.\n\n## Initial Analysis\n{json.dumps(data, indent=2)}'
+            suffix = """
+# Instructions
+0. Take a look at below `Initial Analysis`, based on a recorded trace of the bug. Pay special attention to `IMPORTANT_NOTES`.
```

Comment:
NOTEs: This works!
* It indeed pays very careful attention to `IMPORTANT_NOTES`, which we can now add into analysis data.
* It does state the main problem and propose a plan (again: for self-consistency).